lib/rtsp.c

```diff
@@ -822,6 +861,49 @@ CURLcode Curl_rtsp_parseheader(struct Curl_easy *data, char *header)
       (data->set.str[STRING_RTSP_SESSION_ID])[idlen] = '\0';
     }
   }
+  else if(checkprefix("Transport:", header)) {
```

Comment:
How about making the `Transport:` header parser into a separate function to make things easier to read and follow?

Reply:
I think it is a good idea.